lib/i18n/js/engine.rb

```diff
@@ -3,7 +3,7 @@
 module I18n
   module JS
     class Engine < ::Rails::Engine
-      initializer "i18n-js.register_preprocessor", :after => "sprockets.environment" do
+      initializer "i18n-js.register_preprocessor", after: :engines_blank_point, before: :finisher_hook do
```

Comment:
Ok, little bit more details:

1. Here is where sprockets are attached with Rails. There is no `'sprockets.environment'` mentioned.
https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb

2. Finisher hook is the place which should be used as border. 
http://guides.rubyonrails.org/configuring.html#initializers

https://github.com/rails/sprockets-rails/blob/master/lib/sprockets/railtie.rb#L160